Document all breaking changes. (#846)
telerik/maui-docs: upgrade/overview.md

## Upgrade Using the Telerik NuGet Server

To upgrade the Telerik UI for .NET MAUI components used in your project:

1. Make sure that you have a NuGet feed source with the version you want to upgrade to. This is usually the Telerik NuGet Feed (on [Windows]({%slug telerik-nuget-server%}) or [MacOS]({%slug telerik-nuget-server-mac%})), but you can also use a local feed from the [MSI installer]({%slug maui-getting-started%}) or [PKG installer]({%slug install-pkg%}) or [ZIP archive]({%slug download-product-files%}).

1. Update the version of the `Telerik.UI.for.Maui` package your project references. If you are using a trial version, the package name is `Telerik.UI.for.Maui.Trial`.

## Upgrade Manually Added Assemblies

You can use a [manual installation approach]({%slug installation-approaches%}#manually-using-assembly-references) for referencing the required Telerik UI for .NET MAUI assemblies into your solution. In this case, to upgrade the components to a newer version, you need to replace the assemblies with the assemblies from the latest released version of Telerik .NET MAUI.

> > If you have installed the trial version of Telerik UI for .NET MAUI and want to install the commercial version, you have to remove the trial version first.

Reviewer: If this is a requirement, it shouldn't be a note in this section. It should be the first step in the next section that handles the upgrade from trial to commercial.

## Upgrade from Trial to Commercial

> If you have purchased a license and you need to migrate from the trial package to the licensed version:

Reviewer: ```suggestion
If you have purchased a license and you need to migrate from the trial package to the licensed commercial version of the product:
```

1. Run the application.

## See Also

> - [I Still See Trial Message But Having a Commercial License]({%slug trial-message-commercial-license%})

Reviewer: ```suggestion
- [I Still See the Trial Message after Purchasing a License]({%slug trial-message-commercial-license%})
```
